Classes/Accounts/List/Screens/HomeViewController.swift

```diff
@@ -625,6 +625,25 @@ extension HomeViewController {
             self.triggerBannerCTA(item: item)
         }
     }
+    
+    private func linkInteractors(
+        _ cell: CardAnnouncementCell,
+        for item: AnnouncementViewModel
+    ) {
+        cell.startObserving(event: .close) {
+            [weak self] in
+            guard let self = self else { return }
```

Comment:
Since swift 5.7 you can use short form for unwrapping optionals.
https://www.hackingwithswift.com/swift/5.7/if-let-shorthand